data-processing-lib/python/src/data_processing/test_support/abstract_test.py

```diff
@@ -218,7 +218,7 @@ def __confirm_diffs(
                 # These are binary files.
                 da = DataAccessLocal()
                 f1_bytes = da.get_file(src)
-                f2_bytes = da.get_file(dest)
```

Comment:
should we compare the expected and destination?

Reply:
nop, source and expected